src/Bitcoin/TransactionBuilder.cpp

```diff
@@ -77,8 +77,16 @@ int64_t estimateSegwitFee(const FeeCalculator& feeCalculator, const TransactionP
     return fee;
 }
 
+int extraOutputCount(const SigningInput& input) {
+    int count = int(input.outputOpReturn.size() > 0);
```

Comment:
This returns a boolean if `outputOpReturn` contains elements, is it intended? Maybe make it return `bool` instead, renamed to `hasExtraOutputCount`? And below where `output_size` is calculated cast the boolean to an `int` just for the addition (line 114).

Or remove the function, its code can be inlined and be just as expressive.

Reply:
Github's compare function is a bit off after the rebase - the code is not our contribution.